libs/datamodel/core/tests/config/sources.rs

```diff
@@ -69,7 +69,6 @@ fn must_forbid_env_functions_in_provider_field() {
             url = env("DB_URL")
         }
     "#;
-    std::env::set_var("DB_PROVIDER", "postgresql");
```

Comment:
I guess the test would fail if this line is present?

Reply:
Its hard to make a test fail for that code really, but the test failed with the previous code without it.

Reply:
Right, I will fix that.

Reply:
Fixed @mavilein